bundle/config/target.go

```diff
@@ -37,6 +40,8 @@ type Target struct {
 	RunAs *jobs.JobRunAs `json:"run_as,omitempty"`
 
 	Sync *Sync `json:"sync,omitempty"`
+
+	Permissions []resources.Permission `json:"permissions,omitempty"`
```

Comment:
Top-level target fields need to be manually merged into the main config.

Please add a test under `bundle/tests` that confirms that this happens.